src/Fable.ReactDom.fs

```diff
@@ -36,6 +46,13 @@ module ReactDomBindings =
     #endif
     let ReactDom: IReactDom = jsNative
 
+    #if FABLE_REPL_LIB
+    [<Global("ReactDOM")>]
+    #else
+    [<Import("*", "react-dom/client")>]
```

Comment:
Should this be `*` or `default`? What about the `FABLE_REPL_LIB` bit?

Reply:
You can skip that, we used it for the repl in Fable 2 but we haven't uploaded external libraries yet to the repl for Fable 3.